grammar.rst

```diff
@@ -28,47 +28,38 @@ operator (``//``) broke the :mod:`parser` module.
 Checklist
 ---------
 
-* :file:`Grammar/Grammar`: OK, you'd probably worked this one out :)
+Note: sometimes things mysteriously don't work.  Before giving up, try ``make clean``.
+
+* :file:`Grammar/Grammar`: OK, you'd probably worked this one out. :-)  After changing
+  it, run ``make regen-grammar``, to regenerate :file:`Include/graminit.h` and
+  :file:`Python/graminit.c`.  (This runs Python's parser generator, ``Python/pgen``.
```

Comment:
Closing parenthesis is missing.

Reply:
Resolved in a7cc4fb. Good catch @ilevkivskyi